packages/ember-glimmer/tests/integration/helpers/closure-action-test.js

```diff
@@ -689,10 +689,10 @@ moduleFor('Helpers test: closure {{action}}', class extends RenderingTest {
     });
 
     let OuterComponent = Component.extend({
-      init() {
-        this._super(...arguments);
-        outerComponent = this;
-      },
+      // init() {
```

Comment:
The issue here was that `outerComponent` was unused. I wasn't sure if it was actually supposed to be used in a test somewhere so I commented it out for now. Let me know if I should remove is completely.